code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -45,6 +45,8 @@
 				qdel(src)
 				return
 
+#define CHEMMASTER_BOTTLE_SOUND playsound(src, 'sound/items/pickup/bottle.ogg', 75, 1)
+
```

Comment:
The defines should be placed at the top unless there is a good reason not to do so.